docs: update chapter 5

AndiFr opened the pull request:
Gemeinsamer Branch für Anpassungen in Kaptiel 5 des Abschlusberichtes.

AndiFr committed:
docs: update chapter 5

mizu492 commented:
Mein Vorschlag zu den klassischen Ansätzen ist, die Unterkapitel mit den Erklärungen ganz raus zu nehmen. Ich denke einfach, dass wir die nicht wirklich brauchen (das Modul von Thimm war ja Voraussetzung für das Projekt). Dann geben wir die Vorabversion am Mo ab und schauen was er dazu sagt. Ich kann evtl. ja in der EMail auch wegen genau diesem Punkt nachfragen.
Falls er dann mehr sehen will, fügen wir wieder etwas ein.
Meiner Meinung nach kommen wir sonst nicht auf die 12 Seiten.
Gibt es andere Meinungen dazu?

felLind commented:
Sehe ich wie Anne. Am Ende ist das ein Praktikumsbericht, der zusammenfasst, was wir gemacht haben. Wenn jemand tiefergehende Infos haben will, kann er in die Quellen schauen. Eine Auflistung bzw.  evtl. ein kurzer Satz zu jedem Modell und eher die Motivation warum wir die genommen haben.

felLind committed:
removed sections for traditional ml methods in ch5

mizu492 committed:
Update small additions chap 5

milo-dev98 commented:
> Mein Vorschlag zu den klassischen Ansätzen ist, die Unterkapitel mit den Erklärungen ganz raus zu nehmen. Ich denke einfach, dass wir die nicht wirklich brauchen (das Modul von Thimm war ja Voraussetzung für das Projekt). Dann geben wir die Vorabversion am Mo ab und schauen was er dazu sagt. Ich kann evtl. ja in der EMail auch wegen genau diesem Punkt nachfragen. Falls er dann mehr sehen will, fügen wir wieder etwas ein. Meiner Meinung nach kommen wir sonst nicht auf die 12 Seiten. Gibt es andere Meinungen dazu?

@mizu492 ich find deine Änderungen gut, sollen wir aber bei der Kurzbeschreibung der Klassischen noch so etwas wie : 

 "Detaillierte Beschreibungen zu den Modellen sind in der  Fachliteratur \cite{... }, sowie im Kursskript \textit{Einführung in Maschinelles Lernen} zu finden. "

und dann das Skript auch in die Literatur mit aufnehmen oder ist das eher unnötig ?

mizu492 committed:
Update small changes (transformers part) chap 5

mizu492 commented:
@milo-dev98 ja gute Idee. Soll ich das mit aufnehmen? Ich sitzte gerade an Kap 5.
Ich würde allerdings nicht das Skript zitieren, sondern Fachliteratur (The Elements of Statistical Learning). 
Zumindest in meinem Masterseminar wurde das verboten (Skripte verwenden).

Burak-Oezkan commented:
Zeile 23
> Aus diesem Grund haben wir uns für die Verwendung von Transformer basierten Large Language Model (LLM) entschieden.

Hier passt die Abkürzung mit dem Glossar nicht wirklich, oder? Hier müsste "LLM" im Plural stehen.

mizu492 committed:
Update Referenz für die klassischen Modell 

Ich muss noch die Referenz an sich einfügen, weiß allerdings nicht wie ich references.bib zu dem Pull Request hinzufügen kann?

mizu492 committed:
Update small changes (BERT + DeepSeek)

mizu492 committed:
Update DistilBERT wird mit einem "l" geschrieben

mizu492 committed:
Update Eechtschreibung

AndiFr commented:
> Zeile 23
> 
> > Aus diesem Grund haben wir uns für die Verwendung von Transformer basierten Large Language Model (LLM) entschieden.
> 
> Hier passt die Abkürzung mit dem Glossar nicht wirklich, oder? Hier müsste "LLM" im Plural stehen.

Guter Punkte.
Das können wir aber noch anpassen. Am Ende kommt es darauf an - falls wir Glossar-Definitionen verwenden - wo die erste Verwendung im Text sein wird (also `gls{llm}` verwendet wird), da müssen wir dann prüfen, ob die Form angepasst werden müsste. 
Würde ich persönlich aber auch erst prüfen, wenn alles in den `main` Branch übertragen wurde.

AndiFr committed:
Merge remote-tracking branch 'origin/main' into docs/improvement-report-chapter-5
